datacontract/engines/soda/connections/oracle.py

```diff
@@ -1,10 +1,23 @@
 import os
 
+import oracledb
```

Comment:
bitte den inport in die funktion schieben (oracle ist ja nicht bei allen extras intalliert)